### PR TITLE
Nlc setup

### PR DESCRIPTION
**Pull request by afbenedict**
Set up initial back end 
Returns which module the question is for

### CODE REVIEW

File: src/brutus-api/brutus_api/tasks.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from rq import get_current_job
+
+from brutus_api import app
+from brutus_api import nlp
+import json
+
+from watson_developer_cloud import AuthorizationV1
+
+
+classifierName = "brutus-api"
```
Comment (CtrlC-Root) [inline src/brutus-api/brutus_api/tasks.py]: Should this be a setting?

**Comment by CtrlC-Root** [comment]
Code looks good. The only thing I don't particularly like is requiring an `auth.json` file. An alternative would be to load the credentials through environment variables.

You could store a `watson-credentials.sh` file instead with:

```
export WATSON_USERNAME="XXX"
export WATSON_PASSWORD="XXX"
```

Then load it inside of the `brutus-api-worker.conf` script section before running the app with:

```
. /vagrant/src/brutus-api/watson-credentials.sh
```

Lastly you can then retrieve the settings in `app.py` using `os.getenv`. If you wanted to do local development you could then run the app with your own credentials:

```
export WATSON_USERNAME="..."
export WATSON_PASSWORD="..."
brutus_api --host 0.0.0.0 --port 5000
```

File: src/brutus-api/brutus_api/tasks.py (hunk 1 above)
Comment (afbenedict) [inline src/brutus-api/brutus_api/tasks.py]: yes it probably should be. Do you this this should go in app.py or be an environment variable like the username and password?

Comment (CtrlC-Root) [inline src/brutus-api/brutus_api/tasks.py]: I think an environment variable would be best. That would make it really easy to change when debugging. Although you can make `brutus-api` the default value in the call to `os.getenv()`.

**Comment by CtrlC-Root** [comment]
🚢 🇮🇹

**Pull request merged**